src/apic-extension/HISTORY.rst

```diff
@@ -3,6 +3,22 @@
 Release History
 ===============
 
+1.1.0
+++++++++++++++++++
+Potential Impact: The changes in this release, including the renaming of commands and parameters, may require changes to existing scripts and integrations. Please review the changes carefully and update your code accordingly.
+
+**Updates:**
+* Added example for ``az apic api update`` command.
+* Added examples with system assigned identity for ``az apic create`` and ``az apic update`` commands.
+* [BREAKING CHANGE] Added ``--custom-metadata-only`` parameter to ``az apic metadata export`` command.
+
+**Fixes:**
+* Resolved errors occurring when extracting external documentation from spec in ``az apic register`` command.
```

Comment:
Let's make it more clear about what the error is: Set external document correctly in `az apic register` command.